etna/search/tests/test_forms.py

```diff
@@ -4,93 +4,95 @@
 
 
 class CatalogueSearchFormTest(SimpleTestCase):
-    def test_valid_opening_date_range_values(self):
+    def test_valid_date_range_values(self):
```

Comment:
I was being nice there because you seemed to be upset about having to change the tests again.

I'm using a loop here to ensure that we're testing both ranges in EXACTLY the same way (which I believe is very different to the approach you took in #991). Given those obvious benefits, I don't personally mind the additional request handling.

P.S. It takes a certain sort of person to quote what people have said in one context and throw it at them to try and make a point. I would expect more from you, Allan.

Reply:
As I was not sure what to reply then, and I should not have written it in the way I did.
The intention was only to understand why it was not separated here in this PR while it was asked to sperate it over there, so I had added the lines in context to understand, and not drop it as some back line to you. Sorry. I should not have to put the comments from previous ticket or rephrased it differently without a smiley. Thanks for the explanation of the why for keeping it in same test.